projects/iot-cloud/software/lora2mqtt/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM tiangolo/uwsgi-nginx:python3.6
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY src/ /app
```

Comment:
EOL at EOF

Reply:
If there will be other changes, I will also spend a commit for this `\n`.